app/controllers/katello/api/v2/host_errata_controller.rb

```diff
@@ -3,6 +3,12 @@ class Api::V2::HostErrataController < Api::V2::ApiController
     include Katello::Concerns::FilteredAutoCompleteSearch
     include Katello::Concerns::Api::V2::HostErrataExtensions
 
+    TYPES_FROM_PARAMS = {
```

Comment:
cant you just use https://github.com/Katello/katello/blob/master/app/models/katello/erratum.rb#L9 ?

Reply:
I guess you need a map ??

Reply:
Yeah, for example I need to be able to send a `type` param of just `bugfix`, but the backend still needs to know to search for types matching either `bugfix` or `recommended`.  So passing in the param as-is wouldn't work here.